ignite/engine/events.py

```diff
@@ -93,44 +113,50 @@ def wrapper(engine, event: int) -> bool:
 
         return wrapper
 
+    def __str__(self) -> str:
+        return "<event=%s, filter=%r>" % (self.__class__.__name__, self.filter)
+
 
-class Events(CallableEvents, Enum):
+#
+class Events(CallableEventWithFilter, Enum):
     """Events that are fired by the :class:`~ignite.engine.Engine` during execution.
 
-    Since v0.3.0, Events become more flexible and allow to pass an event filter to the Engine:
+     Since v0.3.0, Events become more flexible and allow to pass an event filter to the Engine:
```

Comment:
The spaces added by your IDE seem to be incorrect, no ? Lines should be aligned with above `"""`